lib/elixir/lib/module.ex

```diff
@@ -1824,22 +1802,11 @@ defmodule Module do
   defp args_count([], total, defaults), do: {total, defaults}
 
   @doc false
-  def check_derive_behaviours_and_impls(env, set, bag, all_definitions) do
+  def check_derive_behaviours_and_impls(env, set, bag) do
```

Comment:
Shall we rename it to:

```suggestion
  def __check_attributes__(env, set, bag) do
```

As it is now mostly about validating certain attributes were set?